exporter/datadogexporter/internal/hostmetadata/metadata.go

```diff
@@ -169,23 +168,18 @@ func RunPusher(ctx context.Context, params exporter.CreateSettings, pcfg PusherC
 		hostMetadata = metadataFromAttributes(attrs)
 	}
 	fillHostMetadata(params, pcfg, p, &hostMetadata)
-
-	// Run one first time at startup
-	if err := pusher.Push(ctx, hostMetadata); err != nil {
-		params.Logger.Warn("Initial host metadata failed", zap.Error(err))
-	} else {
-		params.Logger.Info("Sent initial host metadata")
```

Comment:
Do you want to keep this log?

Reply:
We now have a log each time we push here: https://github.com/DataDog/opentelemetry-mapping-go/blob/8cccaf6f3cfb6e4ad31da562517c3a656f1692d7/pkg/inframetadata/reporter.go#L129-L132